packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -186,6 +186,86 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	async replayFSJournal(events: FilesystemOperation[]) {
 		return replayFSJournal(this.__internal_getPHP()!, events);
 	}
+
+	async downloadWordPressAssets() {
+		await downloadWordPressAssets(this.__internal_getPHP()!);
+	}
+}
+
+async function downloadWordPressAssets(php: PHP) {
+	if (!php.requestHandler) {
+		logger.warn('No PHP request handler available');
+		return;
+	}
+
+	try {
+		const remoteAssetListPath = joinPaths(
+			php.requestHandler.documentRoot,
+			'wordpress-remote-asset-paths'
+		);
+		// If the remote asset list is not available this means that the WordPress static assets are already downloaded.
+		if (!php.fileExists(remoteAssetListPath)) {
+			return;
+		}
+		const wpVersion = await getLoadedWordPressVersion(php.requestHandler);
+		const staticAssetsDirectory =
+			wpVersionToStaticAssetsDirectory(wpVersion);
+		if (!staticAssetsDirectory) {
+			return;
+		}
+		const response = await fetch(
+			[
+				wordPressSiteUrl,
+				staticAssetsDirectory,
+				'wordpress-static.zip',
+			].join('/')
```

Comment:
I'm always concerned about accidentally introducing a double slash in cases like this. We have a join helper for paths in php-wasm/util package, let's use it here

Reply:
Thanks!